docs/examples.rst

```diff
@@ -10,23 +10,7 @@ Notebooks:
 
 .. toctree::
    :maxdepth: 2
+   :glob:
 
-   examples/notebooks/hello-world.ipynb
-   examples/notebooks/binderexample/StatisticalAnalysis.ipynb
-   examples/notebooks/XML_ImportExport.ipynb
-   examples/notebooks/ShapeFactor.ipynb
-   examples/notebooks/multiBinPois.ipynb
-   examples/notebooks/multichannel-coupled-histo.ipynb
-..
-   examples/notebooks/Recast.ipynb
-   examples/notebooks/StatError.ipynb
-   examples/notebooks/example-tensorflow.ipynb
-   examples/notebooks/histogrammar.ipynb
-   examples/notebooks/histosys-pytorch.ipynb
-   examples/notebooks/histosys.ipynb
-   examples/notebooks/importxml.ipynb
-   examples/notebooks/multichannel-coupled-normsys.ipynb
-   examples/notebooks/multichannel-normsys.ipynb
-   examples/notebooks/normsys.ipynb
-   examples/notebooks/pytorch_tests_onoff.ipynb
-   examples/notebooks/tensorflow-limit.ipynb
+   examples/notebooks/*
```

Comment:
I'm not sure we want to add all the notebooks that are under `docs/examples/notebooks/`. We currently only have the notebooks that we test in the notebooks CI being shown as examples and I think that's a good pattern to keep. Also, I think there are some notebooks in there that either need to be overhauled or have huge slabs of text added to them to give better context.

Reply:
Nevermind, I see now this is all controlled through `conf.py`